ledger-core/v2/network/messagesend/service.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/insolar/blob/master/LICENSE.md.
+
+package messagesend
```

Comment:
I've missed this name, it looks realy ugly, it should not be called with verb, so it either MessageSender or Messenger.